pkgs/development/python-modules/crc/default.nix

```
@@ -0,0 +1,22 @@
+{ lib
+, buildPythonPackage
+, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "crc";
+  version = "4.2.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-W1ElEk1127TSXsEHvaU/32zapdY8dH82CWdBNY9r2j4=";
+  };
```

```suggestion
  };
  
  nativeCheckInputs = [
    pytestCheckHook
  ];
```

Because right now I'm seeing 

```
----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

https://github.com/Nicoretti/crc/tree/master/test

Ideally we run unit and integration tests, but not benchmarks.